plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/frontend.ts

```diff
@@ -70,18 +84,24 @@ const getInputData = (
 	}
 
 	const parsedValue = parseInt( inputElement.value, 10 );
-	const parsedMinValue = parseInt( inputElement.min, 10 );
-	const parsedMaxValue = parseInt( inputElement.max, 10 );
-	const parsedStep = parseInt( inputElement.step, 10 );
-
-	const currentValue = isNaN( parsedValue ) ? 0 : parsedValue;
-	const minValue = isNaN( parsedMinValue ) ? 1 : parsedMinValue;
-	const maxValue = isNaN( parsedMaxValue ) ? undefined : parsedMaxValue;
-	const step = isNaN( parsedStep ) ? 1 : parsedStep;
+	const context = getContext< Context >();
+	const productType = context.productType;
```

Comment:
Nit: we could use destructuring to save one line.

```suggestion
	const { productType } = getContext< Context >();
```

Reply:
Fixed in [1600564](https://github.com/woocommerce/woocommerce/pull/59386/commits/160056417057f0250a6c29cf32e539b5c8abd504)